zstd/_generate/gen.go

```diff
@@ -50,6 +50,13 @@ func main() {
 	o.genDecodeSeqAsm("sequenceDecs_decode_amd64")
 	o.bmi2 = true
 	o.genDecodeSeqAsm("sequenceDecs_decode_bmi2")
+
+	exec := executeSimple{copyThreshold: 1024}
```

Comment:
I don't really understand this. Why this limit?

Reply:
I'm assuming that copying larger data (several kB) would be handled better by the Go library.

Reply:
Removed that path.